.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "packages/data-api/scripts/pg-mv-fast-refresh"]
+	path = packages/data-api/scripts/pg-mv-fast-refresh
+	url = git@github.com:beyondessential/pg-mv-fast-refresh.git
```

Comment:
In case you've never used git submodules before (as I had) then what this allows is for git to keep a record of another repo which it will fetch to that directory if requested. This is pretty ideal for us in this situation as we only need to pull down that repo to install the pg-mv-refresh functions, so most of the time we don't actually need it in our project 👍